kiwi/xml_state.py

```diff
@@ -554,6 +556,9 @@ def is_xen_guest(self):
 
         :rtype: bool
         """
+        if self.host_architecture != 'x86_64':
+            # All of Xen only really exists on x86_64
+            return False
         firmware = self.build_type.get_firmware()
         machine_section = self.get_build_type_machine_section()
         if firmware and firmware in Defaults.get_ec2_capable_firmware_names():
```

Comment:
Here is where I would have added the arch check. I but probably is just simpler to avoid any XEN support for non x86 archs if no one is using it or if it doesn't exist a really working tool chain for that yet.

Reply:
This is exactly what I want "No Xen for any other arch than x86_64" You are right there are Xen stuff for other archs but honestly I don't care. I see Xen at a dead end. The only cloud vendor which still uses it is Amazon in EC2 and they also start to move to kvm. They have done the move from Xen para virtualization to Xen host virtualization (full virtualization) in almost all regions.

But I should change the comment line here